src/test/unit/algorithms/TemporalMemoryTest.cpp

```diff
@@ -75,7 +75,10 @@ namespace nupic {
     testNumberOfCells();
     testMapCellsToColumns();
     testWrite();
-    testSaveLoad();
+
+    // Depreciated serialization method (incomplete due to lack of load & save
+    // in Connections and Random classes). Replaced by Cap'n Proto read & write.
+    //testSaveLoad();
```

Comment:
Yes, here's the master commit for it https://github.com/numenta/nupic.core/commit/716e871c61c229d43375b04ec4d9ae79a275c12e#diff-0bc2ea60ba3ca3152541408fc9228b4e